gnucash/ui/gnc-plugin-page-register.ui

```diff
@@ -846,7 +846,7 @@
         <property name="can-focus">False</property>
         <property name="label" translatable="yes">Sche_dule…</property>
         <property name="action-name">GncPluginPageRegisterActions.ScheduleTransactionAction</property>
-        <property name="tooltip-text" translatable="yes">Create a Scheduled Transaction with the current transaction as a template</property>
+        <property name="tooltip-text" translatable="yes">Create a Scheduled Transaction with the current transaction as a template or edit the Scheduled Transaction that current transaction was created by</property>
```

Comment:
Full stop still missing.
I will fix it despite string freeze, because it will be the same string as we already have.

Reply:
Sorry! My Regex didn't fix all of them, because I didn't notice there are <attributes> as well as <properties>. Now I found even more full stops, and fixed them all, I think → #1895.